main.go

```diff
@@ -0,0 +1,72 @@
+
+package main
+
+import (
+    "fmt"
+    "log"
+	"net/http"
+	"io"
+	"encoding/csv"
+	"os"
+)
+
+func getInfo(w http.ResponseWriter, r *http.Request){
+	keys, ok := r.URL.Query()["id"]
+	found := false
+    
+    if !ok || len(keys[0]) < 1 {
+        fmt.Fprintf(w, "Url Param 'id' is missing")
+        log.Println("Url Param 'id' is missing")
+        return
+	}
+	key := keys[0]
+
+	// Open the file
+	csvfile, err := os.Open("data.csv")
+	if err != nil {
+        fmt.Fprintf(w, "Couldn't open the csv file")
+		log.Fatalln("Couldn't open the csv file", err)
```

Comment:
```suggestion
		log.Fatalln("Couldn't open the csv file", err)
		return
```

I think you'd like to return here since the file couldn't be open in this flow.